ontopy/ontology.py

```diff
@@ -1468,18 +1468,25 @@ def closest_common_ancestor(*classes):
             "A closest common ancestor should always exist !"
         )
 
-    def get_ancestors(self, classes, include="all", strict=True):
+    def get_ancestors(
+        self,
+        classes: "Union[List, ThingClass]",
+        common: bool = False,
+        generations: int = None,
+        strict: bool = True,
+    ) -> set:
         """Return ancestors of all classes in `classes`.
-        classes to be provided as list
-
-        The values of `include` may be:
-          - None: ignore this argument
-          - "all": Include all ancestors.
-          - "closest": Include all ancestors up to the closest common
-            ancestor of all classes.
-          - int: Include this number of ancestor levels.  Here `include`
-            may be an integer or a string that can be converted to int.
+        Args:
+            classes: class(es) for which ancestors are desired.
+            common: whether to only return the closest common ancestor.
```

Comment:
```suggestion
            common: whether to only return the closest common ancestor of `classes`.
```

Reply:
There already is a closest_common_ancestor, so no reason to have it here. 

The `common` is equivalent to the include='closest' previously, which would only return the ancestors up to the closest common ancestor. Maybe closest is a better argument?

The reason for having 'common' was that we previously discussed making the to get_ancestors and get_descendants more harmonized. In get_descendants, however, common only returns common descendants. 

maybe we should have the argument `closest` in get_ancestors and `common` in get_descendants and accept that the two functions will not have the same arguments?

Reply:
Thank, now I understand. 

Yes, I think that common is misleading, since the intention is to return the not-common ancestors. I think we should also be more clear in the documentation of the `common` argument.

By the way, `strict` seems to be ignored when `common=True`. The expected behaviour would be to exclude the common ancestor when `strict=True` (i.e. CompositeParticle in the case of `emmo.get_ancestors((onto.Atom, onto.Electron), common=True, strict=True)`.

Reply:
No, strict is only to exlude the classes themselves, i.e. only have true ancestors. __

Reply:
You are right. We should not give more meanings.